src/main/java/com/greencity/pages/SingInPage.java

```diff
@@ -0,0 +1,41 @@
+package com.greencity.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+
+
+public class SingInPage extends BasePage {
+
+    private WebElement emailForm;
+    private WebElement passwordForm;
+    private WebElement singIn;
```

Comment:
You could add our custom elements instead of WebElements from the "pageelements" package for field initialization. But you also need to add the possibility to find elements by "id" if you need it in the future.